tests/testthat/test-missing_argument_linter.R

```diff
@@ -72,11 +72,11 @@ test_that("returns the correct linting", {
 
   expect_lint("switch(a =, b = 1, 0)",
     list(message = rex("Missing argument in function call.")),
-    missing_argument_linter(c()))
+    missing_argument_linter(NULL))
 
   expect_lint("alist(a =)",
     list(message = rex("Missing argument in function call.")),
-    missing_argument_linter(c()))
+    missing_argument_linter(NULL))
 
```

Comment:
That should be a false positive, right? `alist` is special in the sense that it's used explicitly to craft argument lists with or without defaults.